canvas_modules/common-canvas/src/common-properties/controls/expression/expression.jsx

```diff
@@ -320,10 +320,6 @@ class ExpressionControl extends React.Component {
 	}
 
 	render() {
-		const hidden = this.props.state === STATES.HIDDEN;
-		if (hidden) {
-			return null; // Do not render hidden controls
```

Comment:
Because we were returning `null`, `this.editorRef.current = null`. So the codemirror editor for hidden control was always `null` even after the control became visible.

Reply:
Will this cause issues in this case https://github.com/elyra-ai/canvas/issues/2521?